Sources/SecureXPC/Client/XPCClient.swift

```diff
@@ -210,11 +212,13 @@ public class XPCClient {
     /// Sends a request with no message that does not receive a reply.
     ///
     /// - Parameters:
-    ///   - route: The server route which will handle this request.
+    ///   - toRoute: The server route which will handle this request.
     @available(macOS 10.15.0, *)
-    public func send(route: XPCRouteWithoutMessageWithoutReply) async throws {
+    public func send(
+        toRoute route: XPCRouteWithoutMessageWithoutReply
```

Comment:
I would argue that this should be:

```suggestion
        to destination route: XPCRouteWithoutMessageWithoutReply
```

* *Omit needless words.* Every word in a name should convey salient information at the use site.

    More words may be needed to clarify intent or disambiguate meaning, but those that are redundant with information the reader already possesses should be omitted. In particular, omit words that merely repeat type information.

    ```diff
    - public mutating func removeElement(_ member: Element) -> Element?
    
    - allViews.removeElement(cancelButton)
    ```
    In this case, the word Element adds nothing salient at the call site. This API would be better:

    ```diff
    + public mutating func remove(_ member: Element) -> Element?
    
    + allViews.remove(cancelButton) // clearer
    ```
    
* *Name variables, parameters, and associated types according to their roles*, rather than their type constraints.

    ```diff
    - var string = "Hello"
    - protocol ViewController {
    -  associatedtype ViewType : View
    - }
    - class ProductionLine {
    -   func restock(from widgetFactory: WidgetFactory)
    - }
    ```

    Repurposing a type name in this way fails to optimize clarity and expressivity. Instead, strive to choose a name that expresses the entity’s role.
    
    ```diff
    + var greeting = "Hello"
    + protocol ViewController {
    +   associatedtype ContentView : View
    + }
    + class ProductionLine {
    +  func restock(from supplier: WidgetFactory)
    + }
    ```

Reply:
Thanks for the additional context. That makes sense. However, I'm confused by your suggestion:
```swift
to destination route: XPCRouteWithoutMessageWithoutReply
```

As far I'm aware (and I double checked in Xcode) this doesn't appear to be valid? Specifically the middle `destination` token isn't valid syntax. Or am I missing something?

Reply:
oops, I meant:

```suggestion
        to destination: XPCRouteWithoutMessageWithoutReply
```